### PR TITLE
Added OrdnanceSurvey Bing Layer for UK users.

### PR DESCRIPTION
**Pull request by damocov**
Added additional layer for UK users with valid Bing Key, to access the UK Ordnance Survey mapping layer of Bing Maps.

Have added amended version of GWT openlayers bing layers source, with added layer type of ordnancesurvey" to allow traccar files to use the UK only layer.

Amended the MapView.java to add "Bing OSGB" as an option to use the ordnancesurvey layer.

Amended the mobile traccar-mobile.js to use the "Bing OSGB" option and ordnancesurvey layer.

### CODE REVIEW

File: src/main/java/org/gwtopenmaps/openlayers/client/layer/BingType.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ *
+ * @author Giuseppe La Scaleia - CNR IMAA geoSDI Group
+ * @email  giuseppe.lascaleia@geosdi.org
+ */
+public enum BingType {
+
+    ROAD("Road"),
+    HYBRID("AerialWithLabels"),
+    AERIAL("Aerial"),
+    ordnancesurvey("ordnancesurvey");
```
Comment (vitalidze) [inline src/main/java/org/gwtopenmaps/openlayers/client/layer/BingType.java]: Please change enum element name to be in upper-case letters.

File: src/main/java/org/traccar/web/shared/model/UserSettings.java [hunk 2]
```diff
@@ -86,6 +86,7 @@ public double toKnots(double speed) {
         BING_ROAD("Bing Road") { @Override public boolean isBing() { return true; } },
         BING_HYBRID("Bing Hybrid") { @Override public boolean isBing() { return true; } },
         BING_AERIAL("Bing Aerial") { @Override public boolean isBing() { return true; } },
+        BING_OS("Bing OSGB") { @Override public boolean isBing() { return true; } },
```
Comment (vitalidze) [inline src/main/java/org/traccar/web/shared/model/UserSettings.java]: Let's name enum element in a more descriptive way, like `BING_ORDNANCE_SURVEY`, or at least `BING_OSGB`.

**Review by vitalidze — COMMENTED** [review]
Basically looks good, but please remove .save files and there are couple of minor formatting comments.

**Comment by damocov** [comment]
Makes sense, I've made those changes to the branch, but haven't yet had the chance to compile and test

**Comment by vitalidze** [comment]
Thank you for considering these changes. Now please check that it is working on your side and if it does I will merge changes.

**Comment by damocov** [comment]
I've just compiled and confirmed that it is working fine, on both "desktop" and mobile versions.

**Comment by vitalidze** [comment]
Squashed and merged. Thank you!

**Comment by vitalidze** [comment]
Included in latest release, written [news](http://traccar.litvak.su/2016/12/17/Bugfix-Release.html), updated [latest](http://traccar.litvak.su/installation.html) and [demo](http://traccar.litvak.su/demo.html) versions.

**Pull request merged**